sigopt/xgboost/experiment.py

```diff
@@ -181,14 +186,42 @@ def run_experiment(self):
           self.dtrain,
           num_boost_round=num_boost_round_run,
           evals=self.evals,
+          early_stopping_rounds=self.early_stopping_rounds,
           verbose_eval=False,
           run_options=self.run_options,
         )
 
+        # mark early stopping rounds as SigOpt Default
+        if self.early_stopping_rounds == DEFAULT_EARLY_STOPPING_ROUNDS:
```

Comment:
Should also check that `early_stopping_rounds` is not defined in the `experiment_config["parameters"]`, or better yet, forbid users from tuning this parameter altogether.

Reply:
Also want to ensure they aren't passing in same default as us. Seems like 10 might be reasonably common?

https://stackoverflow.com/questions/67202314/how-to-distinguish-default-argument-and-argument-provided-with-default-value

Reply:
good catch.

Reply:
Thanks for catching this, just updated and re-requested review.